src/integrationTest/resources/db/changelog/db.changelog-AM-823.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+  <changeSet id="10" author="nitin.sahu@hmcts.net">
+    <createTable tableName="actor_cache_control">
```

Comment:
Not sure why are we using cache control table here? We can discuss.

Reply:
Hi Nitin, We should be using the same Liquibase scripts as the RAS, so that there is no discrepancy.
The liquibase scripts are being used for Embedded postgres only.